atlascope/core/rest/endpoints/dataset_endpoints.py

```diff
@@ -44,3 +50,21 @@ def create(self, request):
         )
 
         return Response(DatasetSerializer(new_dataset_obj).data, status=status.HTTP_201_CREATED)
+
+    @swagger_auto_schema(request_body=DatasetSubImageSerializer())
+    @action(detail=False, methods=['POST'])
+    def create_sub_image(self, request):
```

Comment:
Following RESTful style this should just be called `subimage`. Then the REST route will be `POST /datasets/{id}/subimage` which we could take to mean "create a subimage dataset from dataset {id}".